account_avatax/views/account_move_view.xml

```diff
@@ -32,6 +35,10 @@
                     name="invoice_doc_no"
                     attrs="{'invisible': [('move_type','!=','out_refund')]}"
                 />
+                <field
+                    name="avatax_amt_line_override"
+                    groups="account_avatax.access_avatax_override_group"
```

Comment:
Shouldn't this field be visible only for refunds?
Doing so, the Python code could be simplified to rely only on this flag, without checking `move_type == "out_refund"`

In fact, I had requests to allow for some lines to use Odoo taxes, for cases where there were special taxes not handled by Avatax. Doing this would be a step towards supporting that.